pages/blog/blog-post.js

```diff
@@ -137,6 +137,47 @@ const SocialShareLink = SocialLink.withComponent('div').extend`
   margin: 0;
 `
 
+const AuthorSection = ({
+  authorTeamMember,
+  authorExternal,
+  authorText,
+}) => {
+  const heading = (authorTeamMember || authorExternal) ? <H4>{authorText}</H4> : ''
+
+  let author = null
```

Comment:
I think the `null` is not necessary. But I guess it is more explicit.

Reply:
Wouldn't it get `undefined` as a default value otherwise?

Reply:
It would. That's the default value, but I don't see how it changes the behaviour of the code.